src/main/java/lotto/domain/LottoMatches.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.constants.Matched;
+
+public class LottoMatches {
+    private final List<Matched> lottoMatches;
+    public LottoMatches(final List<Matched> lottoMatches) {
```

Comment:
```suggestion

    public LottoMatches(final List<Matched> lottoMatches) {
```

전체적으로 답답하지 않게 컨벤션을 맞춰주시면 좋을 것 같습니다 :)

`메서드 사이에 new line 한줄 추가`